### ISSUE TITLE
BrowserView mouse capture region has incorrect bounds.

### ISSUE THREAD

**Issue by ccorcos** [electron/electron#26116]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:** v10.1.4
  * 
* **Operating System:** MacOS Big Sur
  * 

I created a full repro here: https://github.com/ccorcos/electron-browser-view-bounds

```
git clone git@github.com:ccorcos/electron-browser-view-bounds.git
cd electron-browser-view-bounds
npm install 
npm start
```

You can also view the repro example here: https://github.com/ccorcos/electron-browser-view-bounds/blob/main/example.mov

Basically we can resize the view with the window using `view.setAutoResize({ width: true, height: true })` or manually with: 
```
  mainWindow.on('resize', () => {
    const [width, height] = mainWindow.getSize()
    view.setBounds({ x: 0, y: 0, width, height })
  })
```

And the view does properly resize, but the mouse event capture region that gets passed on to the view remains the same size and fits into the bottom right region of the window.

**Comment by codebytere** [comment]
@ccorcos do you know if this worked previously, and if so, in what version?

**Comment by ccorcos** [comment]
@codebytere I just tested it out and it seems this has been broken. For a long time. I created a branch for each version.

```
git checkout electron-11; npm install; npm start # broken
git checkout electron-9; npm install; npm start # broken
git checkout electron-8; npm install; npm start # broken
git checkout electron-7; npm install; npm start # broken
git checkout electron-6; npm install; npm start # broken
git checkout electron-5; npm install; npm start # broken
git checkout electron-4; npm install; npm start # broken
git checkout electron-3; npm install; npm start # broken
```

Electron 2 has an error for some other reason about `app.whenReady` but this is ancient history so I get the feeling that this has always been an issue.
And

**Comment by ccorcos** [comment]
I just added the following which makes it even more clear what the issue is:
```
  document.addEventListener('mouseenter', () => {
    document.body.style.color = 'blue'
  })
  document.addEventListener('mouseleave', () => {
    document.body.style.color = 'red'
  })
```

**Comment by codebytere** [comment]
@ccorcos PR is up :)

**Comment by ccorcos** [comment]
Thanks! 🚂 

I just installed all the dev tooling yesterday (took like 3 hours haha). Are you using VSCode for C++ coding? I really dislike Xcode...
